python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -173,37 +187,32 @@ def rearrange_by_hash(
                 ignore_index,
                 npartitions,
             )
-            for inp in inputs
-        }
 
-        k_list = tuple(range(k))
-        shuffle_split_token = "shuffle-split-" + token
-        split = {  # Get out each individual dataframe piece from the dicts
-            (shuffle_split_token, stage, i, inp): (
-                getitem,
-                (shuffle_group_token, stage, inp),
-                i,
-            )
-            for i in k_list
-            for inp in inputs
-        }
+            _concat_list = []
+            for i in k_list:
```

Comment:
May want to rename `k_list` since it's a tuple

Reply:
Actually now we won't need `k_list` as there is only one usage.